src/librustdoc/clean/mod.rs

```diff
@@ -339,6 +339,14 @@ impl Item {
             _ => String::new(),
         }
     }
+
+    pub fn stable_since(&self) -> Option<String> {
```

Comment:
Can you use `-> Option<&str>` here instead? It should be enough since the string is only ever passed down. Change `s.since.clone()` to `&s.since[..]` too and update the other impacted code.

I think this change looks great otherwise.